contributors.md

```diff
@@ -2,4 +2,5 @@ Contributors file
 =================
 
 bananas24 - teacher
+Hugo - Student
```

Comment:
Hugo, I would like all dashes to be on the same column

Reply:
changes are done